spec/css/functions/special.hrx

```diff
@@ -0,0 +1,24 @@
+<===> options.yml
+---
+:ignore_for:
+- libsass
+
+<===>
+================================================================================
+<===> clamp/numbers/input.scss
+a {b: clamp(0, 1, 2)}
+
+<===>
+================================================================================
+<===> clamp/punctuation/input.scss
+a {b: clamp(@#$%^&*({[]})_-+=|\\:""''<>,.?/)}
+
+<===>
+================================================================================
+<===> clamp/punctuation/input.scss
```

Comment:
This is a duplicate of the previous test

Reply:
Not sure I understand, I don't think the punctuation test is duplicated anywhere?